GNUmakefile

```diff
@@ -89,6 +92,9 @@ install-plugins: ## Build & install LOOPP binaries for products and chains.
 	go install $(GOFLAGS) ./pkg/solana/cmd/chainlink-solana
 	cd $(shell go mod download -json github.com/smartcontractkit/chainlink-starknet/relayer@$(STARKNET_SHA) | jq -r .Dir) && \
 	go install $(GOFLAGS) ./pkg/chainlink/cmd/chainlink-starknet
+	# Set GOPRIVATE to ensure go installs from our repository instead of the public proxy since we're using lightweight git tags.
+	cd $(shell GOPRIVATE=github.com/smartcontractkit/chainlink-internal-integrations go mod download -json github.com/smartcontractkit/chainlink-internal-integrations/aptos/relayer@$(APTOS_RELAYER_GIT_REF) | jq -r .Dir) && \
+	go install $(GOFLAGS) ./cmd/chainlink-internal-integrations
```

Comment:
Hmm, what are you pointing at here?

The plugin entrypoint is here: https://github.com/smartcontractkit/chainlink-internal-integrations/tree/develop/aptos/relayer/cmd/chainlink-aptos